CODEOWNERS

```diff
@@ -1,9 +1,9 @@
 # Mobile
-/onnxruntime/test/testdata/kernel_def_hashes/ @skottmckay @YUNQIUGUO @edgchen1
-/onnxruntime/core/framework/kernel_def_hash_helpers.* @skottmckay @YUNQIUGUO @edgchen1
+/onnxruntime/test/testdata/kernel_def_hashes/ @microsoft/onnxruntime-mobile
+/onnxruntime/core/framework/kernel_def_hash_helpers.* @microsoft/onnxruntime-mobile
 
 # Contrib Ops
-onnxruntime/core/graph/contrib_ops/nhwc_schema_defs.cc @zhanghuanrong @chenfucn @yufenglee @yihonglyu @snnn
-onnxruntime/core/graph/contrib_ops/nchwc_schema_defs.cc @zhanghuanrong @chenfucn @yufenglee @yihonglyu @snnn
-onnxruntime/core/graph/contrib_ops/quantization_defs.* @zhanghuanrong @chenfucn @yufenglee @yihonglyu @snnn
-onnxruntime/core/mlas/** @zhanghuanrong @chenfucn @yufenglee @yihonglyu @snnn
\ No newline at end of file
+onnxruntime/core/graph/contrib_ops/nhwc_schema_defs.cc @microsoft/onnxruntime-contrib-ops
+onnxruntime/core/graph/contrib_ops/nchwc_schema_defs.cc @microsoft/onnxruntime-contrib-ops
+onnxruntime/core/graph/contrib_ops/quantization_defs.* @microsoft/onnxruntime-contrib-ops
+onnxruntime/core/mlas/** @microsoft/onnxruntime-contrib-ops
```

Comment:
They are the people who work on mlas and quantization. We have many other contrib ops that were not created by this team.

Reply:
I see...how about renaming the team to "mlas"? CC @yufenglee

Reply:
Sounds good to me.

Reply:
yes, that's a better name.

Reply:
Updated it.